install_config/aggregate_logging.adoc

```diff
@@ -241,27 +318,8 @@ used for scaling.
 |`*IMAGE_VERSION*`
 |The version for logging component images. For example, setting the version to
 *v1.1* creates *openshift/origin-logging-deployer:v1.1*.
-
```

Comment:
Just noticed IMAGE_PREFIX and IMAGE_VERSION talk about "openshift/origin-" images, even in the OSE build. Should it be ifdef'd to distinguish between the OSE/Origin image locations/names?